src/pages/api/user.ts

```diff
@@ -0,0 +1,18 @@
+import { NextApiRequest, NextApiResponse } from "next";
+
+import { PrismaClient } from "@prisma/client";
```

Comment:
dont call the prisma client directly on each route,
this can potetially open multiple connections

https://github.com/prisma/prisma-examples/blob/latest/typescript/rest-nextjs-api-routes-auth/lib/prisma.ts